dotnet-desktop-guide/framework/winforms/controls/how-to-iterate-through-all-nodes-of-a-windows-forms-treeview-control.md

```diff
@@ -118,6 +120,48 @@ It is sometimes useful to examine every node in a Windows Forms <xref:System.Win
        }  
     ```  
   
+## Non-recursive approach
+
+Recursive code may lead to stack overflow errors or out of memory exceptions due to there being too many frames on the stack.
+
+The following example is an alternate iterative approach to traversing the nodes of the tree using a <xref:System.Collections.Generic.Queue> data structure. This approach does not follow any ordering and only ensures all the nodes are printed. If we wish to follow a pre-order or post-order approach then we can use a <xref:System.Collections.Generic.Stack> data structure to ensure that a node and its children are processed before subsequent nodes.
```

Comment:
Adds the encoded `1 to represent the generic parameter and correctly link the xref

```suggestion
The following example is an alternate iterative approach to traversing the nodes of the tree using a <xref:System.Collections.Generic.Queue%601> data structure. This approach does not follow any ordering and only ensures all the nodes are printed. If we wish to follow a pre-order or post-order approach then we can use a <xref:System.Collections.Generic.Stack%601> data structure to ensure that a node and its children are processed before subsequent nodes.
```

Reply:
Updated the document text to fix the xref as per your reference